pkgs/applications/window-managers/way-cooler/default.nix

```diff
@@ -9,65 +9,62 @@ let
   fakegit = writeShellScriptBin "git" ''
     echo ""
   '';
-  way-cooler = ((callPackage ./way-cooler.nix {}).way_cooler_0_6_2.override {
+  way-cooler = ((callPackage ./way-cooler.nix {}).way_cooler_0_7_0.override {
     crateOverrides = defaultCrateOverrides // {
 
-    way-cooler = attrs: { buildInputs = [ wlc cairo libxkbcommon fakegit ]; };
+    way-cooler = attrs: { buildInputs = [ wlc cairo libxkbcommon fakegit gdk_pixbuf lua5_3 ]; };
     dbus = attrs: { buildInputs = [ pkgconfig dbus_libs ]; };
     gobject-sys = attrs: { buildInputs = [ dbus_glib ]; };
+    gio-sys = attrs: { buildInputs = [ dbus_glib ]; };
+    gdk-pixbuf-sys = attrs: { buildInputs = [ dbus_glib ]; };
+    gdk-pixbuf = attrs: { buildInputs = [ dbus_glib gdk_pixbuf ]; };
```

Comment:
Is it a good idea to specify this in `default-crate-overrides.nix` instead?

Reply:
@ne-sted Indeed, thanks!